src/packages/@ncigdc/containers/RepositoryPage.js

```diff
@@ -265,6 +267,8 @@ export const RepositoryPageQuery = {
           cases {
             aggregations(filters: $filters aggregations_filter_themselves: false) {
               ${CaseAggregations.getFragment('aggregations')}
+            }
+            pieAggregations: aggregations(filters: $filters aggregations_filter_themselves: true) {
```

Comment:
there was also a bug where the label would say 1 files0 or 1cases0 if there was only 1 item in the pie